SocialCareCaseViewerApi/V1/UseCase/WarningNoteUseCase.cs

```diff
@@ -18,5 +20,12 @@ public CreateWarningNoteResponse ExecutePost(CreateWarningNoteRequest request)
             return _databaseGateway.CreateWarningNote(request);
         }
 
+        public GetWarningNoteResponse ExecuteGet(GetWarningNoteRequest request)
+        {
+            var warningNotes = _databaseGateway.GetWarningNotes(request);
+
+            var response = warningNotes.Select(x => x.ToDomain()).ToList();
+            return new GetWarningNoteResponse() { WarningNotes = response };
```

Comment:
If we get rid of the class as per comment on WarningNoteUseCaseTest we can just return a list.

Reply:
Removed GetWarningNoteResponse following discussion